src/index.html

```diff
@@ -4,9 +4,396 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="field">
+      <header>
+        <div class="container">
+          <a class="logo" href="#logo-home">
+            <img
+              class="logo__image"
+              src="images/logo.png" 
+              alt="logo home"
+            >
+          </a>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  href="#apple"
+                  class="nav__link is-active"
+                >
+                  Apple
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#samsung" class="nav__link">
+                  Samsung
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#smartphones" class="nav__link">
+                  Smartphones
+                </a>
+              </li>
+              <li class="nav__item">
+                <a 
+                  href="#laptops&computers"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#gadgets" class="nav__link">
+                  Gadgets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#tablets" class="nav__link">
+                  Tablets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#photo" class="nav__link">
+                  Photo
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#video" class="nav__link">
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+
+      <main class="page">
+        <div class="page-field">
+          <div  class="block page-field__element" data-qa="card">
+            <div class="block__card">
+              <div class="card" data-qa="card">
```

Comment:
a lot of redundant wrappers
```suggestion
        <div class="catalog">
              <div class="card catalog__card" data-qa="card">
```
catalog__card element will be for margins